src/Model/Consignment/BpostConsignment.php

```diff
@@ -214,7 +214,7 @@ public function validate(): bool
         /** @var \MyParcelNL\Sdk\src\Model\MyParcelCustomsItem $item */
         $amount = 0;
```

Comment:
PP: Maybe change the name of the variable to `$totalValue`? The term "amount" is used as "total items" in this case.

Reply:
Done